.npmignore

```diff
@@ -1,5 +1,25 @@
-*
-!dist/**
-!LICENCE
```

Comment:
I know its probably easier to list the gitignore by first excluding everything and then only picking what you want in the package but this is the way Projen generated it. Projen adds does it by listing it out one by one. I couldn't figure out a way to override this. By the way the lib directory is like the old dist directory, some names for directories were switched around.